packages/eslint-typescript/prettier.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Once we publish the prettier config, we can extend from `"@arbitrum/prettier-config"` directly